metadata-models/src/main/pegasus/com/linkedin/schema/KafkaSchema.pdl

```diff
@@ -9,10 +9,20 @@ record KafkaSchema {
    * The native kafka document schema. This is a human readable avro document schema.
    */
   documentSchema: string
+  
+  /**
+   * The native kafka document schema type. This can be AVRO/PROTOBUF/JSON.
+   */
+  documentSchemaType: optional string
 
   /**
    * The native kafka key schema as retrieved from Schema Registry
    */
   keySchema: optional string
 
+  /**
+   * The native kafka key schema type. This can be AVRO/PROTOBUF/JSON.
+   */
+  keySchemaType: optional string
```

Comment:
I have two reasons for adding these two fields:
1. More metadata clarification: If any new user see the kafka ingested metadata, he/she will not be able know exactly which type of schema is associated with topic. As I am new to this, even I felt the same.
2. To set top-level doc field: The task was to set top-level doc field as description of dataset only if schema type is AVRO. So as we are generating dataset properties at outer function i.e. _extract_records, we will need schema type at outer function for adding condition. Hence I added those fields.

Alternatives:
We can have separated functions as get_description() in kafka.py. But this will lead to adding same code and calling same metadata fetching APIs again.